src/App.css

```diff
@@ -1,3 +1,38 @@
 .app {
   text-align: center;
-}
\ No newline at end of file
+}
+
+/* ul:empty + .empty-state {
+  display: block;
+} */
+
+ul {
+list-style: armenian;
+/* width: 250px; */
+text-align: left;
+}
+
+nav {
+  /* font-family: 'Reggae One', cursive; */
+  font-family: 'Press Start 2P', cursive;
+  display: flex;
+  justify-content: space-between;
+  background-color: rgb(123, 235, 220);
+  font-size: larger; 
+  width: 90%;
+  margin: auto;
+  /* word-spacing: 1em; */
```

Comment:
Try to remove unused code! Your future managers will notice who they don't have to clean up after...